### PR TITLE
specified difference between #3 and #4 examples

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix Minor Copy editing is needed for this PR. Thanks.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainController.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Description
 
 -----------
 
-Get a global catalog in the current forest using Discovery.
+Get a global catalog in the current forest using Discovery. Domain type id is used in this scenario. Possible values are 1 = PrimaryDC, 2 = GlobalCatalog, 3 = KDC, 4 = TimeService , 5 = ReliableTimeService or 6 = ADWS.
```
Comment (nenonix) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainController.md]:
```suggestion
Get a global catalog in the current forest using Discovery. Domain type id is used in this scenario. Possible values are 1 = PrimaryDC, 2 = GlobalCatalog, 3 = KDC, 4 = TimeService, 5 = ReliableTimeService or 6 = ADWS. 
```

**Comment by Malind19** [comment]
@e0i Copy edits applied

**Comment by e0i** [comment]
Issue reference: #546

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Please review the final formatting changes as revised by the PR author.

_(Last issue in fourth package)_

**Comment by get-itips** [comment]
Something is not right with the live version of this article, which is, based on the folders in the repo, the Windows Server 2012 one.

If you go to https://docs.microsoft.com/en-us/powershell/module/addsadministration/get-addomaincontroller?view=win10-ps you get the WinServ2016 version of this cmdlet.

If you want to change it to Windows Server 2012 using the version selector, you will receive a message like this:  _The requested page is not available for Windows Server 2012 PowerShell. You have been redirected to the newest product version this page is available for._

If you edit that page, you are not editing the one that is under the folder docset/winserver2012-ps/activedirectory but /docset/windows/addsadministration/

So I think, besides the markdown is in the repo, it isn't published or accesible through Docs.

Maybe we need to fix that before.

Talking specifically about the document, It does not seem well formatted:

image

but I think these "description" were present before @Malind19 commits

**Comment by dariomws306** [comment]
Agree with @get-itips!

**Comment by e0i** [comment]
@tseward Any additional comments?

**Comment by tseward** [comment]
I'm good with the change but if we could get the Descriptions cleaned up as @get-itips notes, that would be great.

**Comment by e0i** [comment]
@Malind19 Please follow-up with suggested revisions for this PR.

**Pull request merged**